lib/csa/ccm/matrix.rb

```diff
@@ -108,27 +133,34 @@ def row(i)
     Row.new(worksheet[i])
   end
 
+  def self.get_start_row(version)
+    case version
+    when '1.0'
+      2
+    when '1.1'
+      3
+    when '3.0.1'
+      4
+           end
```

Comment:
Stray whitespace not aligning in beginning?

Reply:
Done